apex-city.md

```diff
@@ -42,7 +42,7 @@ Whether characters are interested in viewing scholarly exhibits of the latest fi
 Characters bringing large amounts of valuable treasure from their latest adventure can find the banking services they need in the Upper Drakesdown ward. The dwarven House Axforge operates the largest bank in the city, the Axforge Bank of Apex, located at the top of Beryl Tower. The Gellidus district of Upper Drakesdown features nine other banks, as well as a number of moneychangers, lenders, and providers of other financial services.
 
 ### Setting up Shop
-Characters may wish to purchase a business or other space in the city. A poor shop (for trades or services) costs 2d6x100 silver, including rough shelves, a sales counter, and a back room for storage. An average shop costs 2d6x1000 silver, including polished wood floors, handsome shelves, a picture window, and glass cases. An upscale shop costs 4d6x1000 silver and has marble floors, locked display cases, leather chairs, and various other luxuries.
+Characters may wish to purchase a business or other space in the city. A poor shop (for trades or services) costs 2d6x100 silver, including rough shelves, a sales counter, and a back room for storage. An average shop costs 2d6x1000 silver, including polished wood floors, handsome shelves, a picture window, and glass cases. An upscale shop costs 4d6x1000 silver and has marble floors, locked display cases, leather chairs, and various other luxuries. In addition, all businesses must petition the city council for a Writ of Charter before beginning business operations.
```

Comment:
Not all businesses require a Writ of Charter! You require a writ if your business intends to do any import or export of goods outside the city. A local apothecary who buys their supply of curatives from other merchants *in the city* has no real need of a writ. A shipping company that imported some of the herbs from abroad and in turn takes raw and finished goods outside of the city for sale requires a Writ of Charter. Many of the larger businesses do not subcontract out their shipping, instead opting to control their own supply chains. For example the Apex Mining Guild understandably does not trust third-party operators with the movement and delivery of their various precious metals and ores, and has their own transportation, and thus a Writ of Charter.

Reply:
And fixed.